ocean_lib/assets/test/test_utils.py

```diff
@@ -107,3 +128,16 @@ def test_add_trusted_algorithm_no_compute_service(publisher_ocean_instance, meta
             algorithm_ddo.did,
             publisher_ocean_instance.config.metadata_cache_uri,
         )
+
+
+def test_fail_generate_trusted_algo_dict():
+    """Tests if all the parameters for generating a trusted algorithm dict are None and if teh scope of this function
+    throws an AssertionError."""
+    try:
+        _ = generate_trusted_algo_dict(None, None, None)
```

Comment:
please remove the `_ ` assessment. Using `_` only works when you are e.g. assigning a tuple and don't want to use one of the elements. If you are explicitly using the underscore you are saying "I will not use this variable". So why assign it in the first place?